platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformWiring.java

```diff
@@ -224,12 +224,24 @@ public PlatformWiring(
         stateSnapshotManagerWiring =
                 new ComponentWiring<>(model, StateSnapshotManager.class, config.stateSnapshotManager());
         stateSignerWiring = new ComponentWiring<>(model, StateSigner.class, config.stateSigner());
-        transactionHandlerWiring = new ComponentWiring<>(model, TransactionHandler.class, config.transactionHandler());
+        transactionHandlerWiring = new ComponentWiring<>(
+                model,
+                TransactionHandler.class,
+                config.transactionHandler(),
+                data -> data instanceof ConsensusRound consensusRound
```

Comment:
I wonder of what other types can `data` be here than `ConsensusRound`?

Reply:
`PlatformWiring.wire()`:
```
        runningEventHashOverrideWiring
                .runningHashUpdateOutput()
                .solderTo(transactionHandlerWiring.getInputWire(TransactionHandler::updateLegacyRunningEventHash));
```
sends `RunningEventHashOverride` to the input wire.